src/fragmentarium/ui/display/DisplayLineNumbers.tsx

```diff
@@ -0,0 +1,41 @@
+import { Line, LineNumber, LineNumberRange, TextLine } from '../../domain/text'
+import React from 'react'
+
+function formatLineNumber(lineNumber: LineNumber): string {
+  return `${lineNumberToString(lineNumber)}`
+}
+
+function formatLineNumberRange({ start, end }: LineNumberRange): string {
+  return `${lineNumberToString(start)}-${lineNumberToString(end)}`
+}
+function lineNumberToString({
+  hasPrime,
+  number,
+  prefixModifier,
+  suffixModifier
+}: LineNumber): string {
+  const prefixMod = prefixModifier ? prefixModifier + '+' : ''
+  const prime = hasPrime ? '′' : ''
+  const suffixMod = suffixModifier ? suffixModifier : ''
+  return `${prefixMod}${number}${prime}${suffixMod}`
+}
+
+const lineNumberTypeToString = {
+  LineNumber: formatLineNumber,
+  LineNumberRange: formatLineNumberRange
+}
+function chooseLineNumberType(
+  lineNumber: LineNumber | LineNumberRange
+): string {
+  const lineNumberToString = lineNumberTypeToString[lineNumber.type as string]
```

Comment:
This hides the `lineNumberToString` defined above. It could be confusing to the reader which function is called below.

Reply:
i renamed the const to `createFormattedLineNumbers`